### ISSUE TITLE
Request Roles for usagov admin cms

### ISSUE THREAD

**Issue by scottqueen-bixal** [GSA/px-bears-drupal#731]
**Create Roles for the following people during Integration**
Gene Chi
Scott Queen
Diego Cobb
Cindy Fong
Ernie Deb

**Comment by EnriqueOrt** [comment]
Email sent out to Amy/Russel on 11/9 - reply has some questions that will need to be reviewed.

Thanks for sending this over!  Amy can go into the details more than me, but some general thoughts:

a. I think before we can figure out timing, we should finish making sure the module works successfully in Drupal 10.  Have you folks been able to do that yet?

b.Also, what would you think about signing up for a Web Ops Demo session to go through how the module works?  I think seeing it in action would be super helpful.  If you folks are game for that, please reach out to [@Isabel Laurenceau - QQGA](mailto:isabel.laurenceau@gsa.gov) as she coordinates those.

After talking to Dave and Amy, I think once everything is working in Drupal 10, Amy will be able to ask a ISAGov engineer to work with your team directly on getting things merged into the USAGov code base and installed onto dev, tested, and then onto Stage and Prod.  
I think that will also let us better answer the question of what the process will be to install new versions and hot fixes, but I suspect it'll be a similar model as this process.

**Comment by scottqueen-bixal** [comment]
a. I think before we can figure out timing, we should finish making sure the module works successfully in Drupal 10. Have you folks been able to do that yet?

yes, you can find a working version on the latest DB and code here, https://benefit-finder-waf-main.app.cloud.gov/benefit-finder/death

b.Also, what would you think about signing up for a Web Ops Demo session to go through how the module works? I think seeing it in action would be super helpful. If you folks are game for that, please reach out to [@Isabel Laurenceau - QQGA](mailto:isabel.laurenceau@gsa.gov) as she coordinates those.

Infra - Fatma
Backend + Drupal Module - Gene
JS app - Scott
Testing - Nehemiah
